api/messages/messages_views.py

```diff
@@ -16,19 +16,22 @@
 @bp.route("/public")
 def public():
     return {
+        "metadata": get_public_message().metadata,
```

Comment:
Why do this if we already have a message with that same structure. Wouldn't be the same to jus return the result of `get_public_message()`? 🤔 

```py
def public():
    return get_public_message()
```